telegram/__init__.py

```diff
@@ -92,7 +93,6 @@
 from .shippingquery import ShippingQuery
 from .webhookinfo import WebhookInfo
 from .gamehighscore import GameHighScore
-from .videonote import VideoNote
```

Comment:
nitpick:
why remove here and add above? (I prefer the minimal change in diff...)